TarkovMapOverlay/MainWindow.xaml

```diff
@@ -6,12 +6,10 @@
         xmlns:local="clr-namespace:TarkovMapOverlay"
         xmlns:sliderMenu="clr-namespace:SliderMenu"
         mc:Ignorable="d"
-        Title="Tarkov Map Overlay" Height="450" Width="800" 
-        WindowStyle="None"
+        Title="Tarkov Map Overlay" Height="450" Width="800"
         BorderBrush="Black"
         BorderThickness="1"
-        Background="Black"
-        AllowsTransparency="True">
```

Comment:
I think that removing `AllowsTransparency` has broken the ability to make the background transparent or to change to opacity. The rest looks good to me and I tested the saving and loading which works well. So just fix this part and I will merge and make a release. Thanks!

![be5fcc2613a3088158cdf2896cb38766](https://user-images.githubusercontent.com/1903284/107186466-a06ead80-69e4-11eb-962b-d9ad48372867.png)

Reply:
oh sorry for that, that was a mistake. i didn't notices that i changed something there.